### PR TITLE
Get started tutorial for experiments

### LINKED ISSUE #2189: Section for dvc exp in https://dvc.org/doc/start/experiments
Add a section for `dvc exp` in the Get Started guide

### PR DESCRIPTION
**Pull request by dberenbaum**
Fix #2189 

Initial draft is ready, and I'd be interested to get feedback, especially about the general structure. I can open a PR in https://github.com/iterative/example-repos-dev once the docs are closer to final.

### CODE REVIEW

**Comment by jorgeorpinel** [comment]
I deployed this to https://dvc-org-experiments-n7elh75kje.herokuapp.com/start.

@dberenbaum for future ref. it's preferred to push a branch to the upstream in this repo so that these review apps are deployed automatically (and it's easier to take over if needed). Thanks

File: content/docs/sidebar.json [hunk 1]
```diff
@@ -53,10 +53,13 @@
         }
       },
       {
-        "slug": "experiments",
+        "slug": "ml-pipelines",
```
Comment (jorgeorpinel) [inline content/docs/sidebar.json:56]: Just looking at the slugs and titles. How do we differentiate "Data Pipelines" from "ML Pipelines"? ML is data driven so it at first glance it just looks like maybe it'll be a more specific version of the previous chapter.

File: content/docs/start/index.md [hunk 2]
```diff
@@ -67,6 +67,10 @@ by one in the next few pages:
   data artifacts are built, and provide an efficient way to reproduce them.
   Think "Makefiles for data and ML projects" done right.
 
-- [**Experiments**](/doc/start/experiments) attach parameters, metrics, plots.
-  You can capture and navigate experiments without leaving Git. Think "Git for
-  machine learning".
+- [**ML pipelines**](/doc/start/ml-pipelines) attach parameters, metrics, plots.
+  You can capture, navigate, and evaluate ML projects without leaving Git. Think
+  "Git for machine learning".
```
Comment (jorgeorpinel) [inline content/docs/start/index.md:72]: Comparing their descriptions in the index, ML pipelines looks like it'll be a continuation of Data pipelines, as a way to split the old content perhaps? But wasn't this PR about experiments? (Haven't looked at the content yet.)

**Review by jorgeorpinel — COMMENTED** [review]
First impressions on structure. Besides the comments below, I'm not sure about using the term "pipeline" for 2 possibly different things since everywhere else there's just one kind of pipeline (stages in dvc.yaml). So hopefully most of my confusion so far has to do with that term 🙂

**Comment by dberenbaum** [comment]
> First impressions on structure. Besides the comments below, I'm not sure about using the term "pipeline" for 2 possibly different things since everywhere else there's just one kind of pipeline (stages in dvc.yaml). So hopefully most of my confusion so far has to do with that term 🙂

I should have provided more context here. The structure is:

```console
$ git mv experiments.md ml-pipelines.md
$ git add experiments.md
```

Not sure what happened, but I seem to have messed this up so that Git is tracking changes between the new and old `experiments.md` instead of `experiments.md` -> `ml-pipelines.md`. There are some changes to `ml-pipelines.md`, but the bulk of this PR is the new `experiments.md`.

**Comment by dberenbaum** [comment]
> First impressions on structure. Besides the comments below, I'm not sure about using the term "pipeline" for 2 possibly different things since everywhere else there's just one kind of pipeline (stages in dvc.yaml). So hopefully most of my confusion so far has to do with that term 🙂

Fair point. Let me know any renaming or restructuring suggestions!

To give my thought process here, the last section of the existing tutorial focused on adding ML-specific functionality to the existing pipeline, namely metrics. Looking at it again, I think this section could be called "Metrics" since that is really the focus of the entire section.

**Comment by jorgeorpinel** [comment]
> Git is tracking changes between the new and old experiments.md instead of experiments.md -> ml-pipelines.md

Ah yeah `git mv` is just a shorthand for `git rm` + `git add` I think. I.e. it doesn't guarantee Git will recognize the change as a rename. I think the way to guarantee it is to do the move in one commit and the edits in another, but going back to do that now may be tricky...

> There are some changes to ml-pipelines.md

If you want you could add some comments in [content/docs/start/ml-pipelines.md](https://github.com/iterative/dvc.org/pull/2191/files?file-filters%5B%5D=.md&file-filters%5B%5D=.svg#diff-3fafa989b9d5ffab0e54e4125241fb21d08a94b0f2aaa1aaa0fc46b4a5c13bbf) to point out those few changes, + we can eyeball it against https://dvc.org/doc/start/experiments so no huge deal prob.

> the last section of the existing tutorial focused on adding ML-specific functionality

True, at least up to **Comparing experiments**. Maybe that part could be moved to the next page? Not sure, let's see what @shcheklein thinks (he wrote the current GS).

> Let me know any renaming or restructuring suggestions!

Following your logic I'd go for something like "Tools for DS/ML" or even a very specific "Parameters and Metrics".

**Comment by dberenbaum** [comment]
Closing this and opening #2195 to fix diff readability.

**Pull request closed without merging**